code_samples/api/migration/src/Command/MigrationCommand.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Command;
+
+use Ibexa\Platform\Migration\MigrationService;
+use Ibexa\Platform\Migration\Repository\Migration;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class MigrationCommand extends Command
+
+{
+    private $migrationService;
+
+    public function __construct(MigrationService $migrationService, ?string $name = null)
+    {
+        $this->migrationService = $migrationService;
+        parent::__construct($name);
+    }
+
+    protected function configure()
+    {
+        $this->setName('doc:migration');
+    }
```

Comment:
When above `protected static $defaultName` is used, the following becomes unnecessary:

```suggestion
    public function __construct(MigrationService $migrationService)
    {
        $this->migrationService = $migrationService;
    }
```